functions/src/export-csv.ts

```diff
@@ -93,8 +93,11 @@ export async function exportCsvHandler(
     await Promise.all(
       snapshot.docs.map(async doc => {
         const loi = doc.data();
-        if (!(ownerIdFilter ? loi[9] === 2 && loi[5] === ownerIdFilter : true))
-          return;
+        if (ownerIdFilter) {
```

Comment:
Also, I think this is still a bit confusing.. maybe we want something like this:

```ts
if (canViewAll || loiOwnerId === userId) {
  return extractLoiProperties(loi);
} else {
  return undefined;
}
```

Then we don't need `ownerIdFilter` at all